### PR TITLE
Update PrivilegedIdentityManagment (PIM) beta document

### PR DESCRIPTION
**Pull request by mingtm**
Added more API examples for PrivilegedOperationEvents and PrivilegedRoleAssignments APIs.

### CODE REVIEW

File: api-reference/beta/api/privilegedapproval_get.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Retrieve the properties and relationships of privilegedapproval object.
 ### Prerequisites
 The following **scopes** are required to execute this API: _Directory.AccessAsUser.All_
 
+The tenant needs to be registered to PIM. Otherwise, HTTP 403 Forbidden error will be returned.
```
Comment (Lauragra) [inline api-reference/beta/api/privilegedapproval_get.md]: FYI, we are going to change "Prerequisites" to "Permissions" globally throughout the content, and updating the boilerplate content to provide more specific information about permissions. We should think about how to present this information, given that it is not related to permissions. Would this information, for example, go in the Response section?

File: api-reference/beta/api/privilegedoperationevent_list.md [hunk 2]
```diff
@@ -28,7 +30,7 @@ Do not supply a request body for this method.
 If successful, this method returns a `200 OK` response code and collection of [privilegedOperationEvent](../resources/privilegedoperationevent.md) objects in the response body.
 ## Example
 ##### Request 1
-Here is an example of the request.
+Here is an example of the request to get all audit events:
```
Comment (Lauragra) [inline api-reference/beta/api/privilegedoperationevent_list.md]: Let's change the H2 here to "Examples" (plural) since we are providing multiple examples. It might also be helpful to have H3s (###) with headings for the specific examples, such as "Get all audit events example", and so on. Then each H3 can have a Request and Response section (no need to number).

Also, please change the introductory text to use this construction: "The following example shows a request to get all audit events."

File: api-reference/beta/api/privilegedoperationevent_list.md [hunk 3]
```diff
@@ -82,39 +85,204 @@ Content-length: 547
...
+```
+##### Request 3
+Here is an example to get the audit events for the operations of self role activation and makePermanent. In this case, ``requestType`` value is ``Activate``.
+
+<!-- { "blockType": "request" } -->
+```http
+GET https://graph.microsoft.com/beta/privilegedOperationEvents?$filter=requestType%20eq%20'Activate'
+```
+##### Response 3
+Here is an example of the response. Note: The response object shown here may be truncated for brevity. All of the properties will be returned from an actual call.
```
Comment (Lauragra) [inline api-reference/beta/api/privilegedoperationevent_list.md]: For the responses, use the construction "The following example shows the response."

File: api-reference/beta/api/privilegedroleassignment_list.md [hunk 4]
```diff
@@ -23,16 +25,16 @@ Do not supply a request body for this method.
 ## Response
 If successful, this method returns a `200 OK` response code and collection of [privilegedRoleAssignment](../resources/privilegedroleassignment.md) objects in the response body.
 ## Example
-##### Request
-Here is an example of the request.
+##### Request 1
+Here is an example to get all role assignments:
```
Comment (Lauragra) [inline api-reference/beta/api/privilegedroleassignment_list.md]: See my earlier comments - add H3s with descriptive titles for the examples, and revise the introductory text.

File: api-reference/beta/api/privilegedapproval_get.md (hunk 1 above)
Comment (Lauragra) [inline api-reference/beta/api/privilegedapproval_get.md]: Global: Revise as "Otherwise, the HTTP 403 Forbidden status code will be returned."

Comment (mingtm) [inline api-reference/beta/api/privilegedapproval_get.md]: moved the new requirement to Response section.

Updated the sentence.

File: api-reference/beta/api/privilegedoperationevent_list.md (hunk 2 above)
Comment (mingtm) [inline api-reference/beta/api/privilegedoperationevent_list.md]: done

File: api-reference/beta/api/privilegedoperationevent_list.md (hunk 3 above)
Comment (mingtm) [inline api-reference/beta/api/privilegedoperationevent_list.md]: done

File: api-reference/beta/api/privilegedroleassignment_list.md (hunk 4 above)
Comment (mingtm) [inline api-reference/beta/api/privilegedroleassignment_list.md]: fixed
